css/layout.css

```diff
@@ -31,4 +31,17 @@
   .column-half {
     width: 50%;
   }
+
+  .row {
+    display: flex;
+    flex-wrap: nowrap;
+  }
+
+  textarea {
+    padding-right: 22.5rem;
+  }
+
+  .code-journal {
+    padding-left: 1rem;
+  }
```

Comment:
These should be moved over to `styles.css`. Only CSS that gets applied to your col / row / container class should go here, or utility classes